packages/generator/src/parser/service-parser.ts

```diff
@@ -132,6 +135,9 @@ function transformServiceMetadata(
     edmxPath: serviceDefinitionPaths.edmxPath
   };
 
+  const transformFunctionImports = isV2Metadata(metadata.edmx)
```

Comment:
I am really looking forward to refactor this long and messy function.

Reply:
yes...